src/parser/jobs/rdm/GeneralCDDowntime.js

```diff
@@ -17,7 +17,7 @@ export default class GeneralCDDowntime extends CooldownDowntime {
 
 	constructor(...args) {
 		super(...args)
-		super.setAbilityList(TRACKEDCDS)
-		super.setDownTimeDuration(DOWNTIME_OK_TIME)
+		super.trackedcds = TRACKED_COOLDOWNS
```

Comment:
`trackedCds`, or even preferably, `trackedCooldowns`

Additionally, no need to do this in the constructor. I'd perfer it follow how About modules handle it.